lib/models/ObjectMD.js

```diff
@@ -352,6 +356,46 @@ class ObjectMD {
         return this._data['content-md5'];
     }
 
+    /**
+     * Set content-language
+     *
+     * @param {string} contentLanguage - content-language
+     * @return {ObjectMD} itself
+     */
+    setContentLanguage(contentLanguage) {
+        this._data['content-language'] = contentLanguage;
+        return this;
+    }
+
+    /**
+     * Returns content-language
+     *
+     * @return {string} content-language
+     */
+    getContentLanguage() {
+        return this._data['content-language'];
+    }
+
+    /**
+     * Set Creation Date
+     *
+     * @param {string} creationTime - Creation Date
+     * @return {ObjectMD} itself
+     */
+    setCreationTime(creationTime) {
+        this._data['creation-time'] = creationTime;
+        return this;
+    }
+
+    /**
+     * Returns Creation Date
+     *
+     * @return {string} Creation Date
+     */
+    getCreationTime() {
```

Comment:
Is there any expectation regarding compatibility with "old" metadata? I.e. do we need e.g. to fallback to `last-modified` if `creation-time` does not exist in the loaded metadata, or is it fine to return `undefined` in such case?

Reply:
Good question, I don't think this has been discussed

Reply:
In Mystique (R.I.P) we had `creation-time` fall back to `last-modified`

Reply:
I've added logic to fallback to `last-modified` if `creation-time` is not set